charts/matrix-stack/ci/quick-setup-certificates-pg-external-values.yaml

```diff
@@ -22,6 +22,11 @@ matrixAuthenticationService:
     port: 5432
     sslMode: prefer
     user: your-matrix-auth-service-user
+matrixRTC:
+  enabled: false
```

Comment:
This looks a bit odd. I guess `matrixRTC` hasn't been setup in `quick-setup-all-enabled.yaml`